src/main/scala/swiss/dasch/api/ProjectsEndpoints.scala

```diff
@@ -225,6 +225,16 @@ final case class ProjectsEndpoints(base: BaseEndpoints) {
     .out(header(HeaderNames.ContentDisposition, "attachment; filename=mapping.csv"))
     .tag("bulk-ingest")
 
+  val postBulkIngestUpload = base.secureEndpoint.post
+    .in(projects / shortcodePathVar / "bulk-ingest" / "upload")
```

Comment:
It's not ingesting it the same way ingest is ingesting it, it's only the upload part, that's why I called it this.

It may not appeal to the eye completely, but I think it makes sense.

Reply:
Yes, this makes sense `upload` rather than `ingest.

Reply:
Oops, didn't see the comment, hadn't refreshed. Implementing now.